lib/client/api_login_test.go

```diff
@@ -507,9 +508,26 @@ func newStandaloneTeleport(t *testing.T, clock clockwork.Clock) *standaloneBundl
 	require.NoError(t, err)
 	user.AddRole(role.GetName())
 
+	makeDataDir := func() string {
+		tempDir := t.TempDir()
```

Comment:
Why not using os.Tempdir directly?

Reply:
That was my first iteration, but it seems to break a bunch of tests. I started going into that rabbit hole but it's more than I want to tackle right now.

Reply:
Some of it has to do with invocations os os.MktempDir assuming certain temp paths exist, some other failures are stranger even, so I didn't get into the root for those.

Reply:
os.TempDir just returns the os temp root directory. All tests will use /tmp instead of creating a temp directory per test.

Can we replace it with os.MkdirTemp or os.MkdirAll and we generate a uuid, hash it and truncate its size

Reply:
And os.RemoveAll will interfere with other tests because their temp dirs are randomly wiped out since you use the temp root for

Reply:
> os.TempDir just returns the os temp root directory

🤦 

Yeah that explains a lot. Let me fix it.

Reply:
This should be good now. Well spotted!